addons/mail/models/mail_thread.py

```diff
@@ -152,6 +152,20 @@ class MailThread(models.AbstractModel):
         help="Number of messages with delivery error")
     message_attachment_count = fields.Integer('Attachment Count', compute='_compute_message_attachment_count', groups="base.group_user")
 
+    partner_fields = fields.Json(compute="_compute_partner_fields")
```

Comment:
Those fields are useless. Moreover they bloat the model and don't hold any record-related value. Just call the model methods directly in _thread_to_store. In the end computed fields just add an extra layer that is not necessary here.

Reply:
This is indeed simpler